importer/importer.go

```diff
@@ -149,6 +154,20 @@ func (i *Importer) fetcher() URLFetcher {
 	return i.Fetcher
 }
 
+// preserveRoot takes an original path and an alteration of that path
+// and returns the altered version with a double slash at the start if
+// the original path starts with a double slash. This is needed as
+// certain functions of the "path" package cleans the paths they return
+// which remove the leading double slash. As that as special meaning
+// to this imported, we need to preserve that prefix.
```

Comment:
Typo?

Reply:
Yup. There was another one and some strange pluralisation that I also fixed in that paragraph.

All fixed.